
=== STDERR ===
Exception in thread "main" util.Assert$TLCRuntimeException: TLC writes its files to a directory whose name is generated from the current time.
This directory should be /workspaces/FormaLLM/outputs/generated/states/25-06-21-03-51-50, but that directory already exists.
Trying to run TLC again will probably fix this problem.
	at util.Assert.check(Assert.java:91)
	at util.FileUtil.makeMetaDir(FileUtil.java:252)
	at tlc2.TLC.handleParameters(TLC.java:958)
	at tlc2.TLC.main(TLC.java:246)
